.github/workflows/main.yaml

```diff
@@ -126,21 +147,28 @@ jobs:
     - name: Build Python
       run: bash ./pythia build
 
-    - uses: actions/upload-artifact@v2
-      with:
-        name: pythia-macos
-        path: dist/**/*.tar.gz
-
     - name: Test Python
       run: bash ./pythia test
 
+    - name: Upload testing package
+      env:
+        BIN_PRIV_KEY: ${{ secrets.SFTPPLUS_BIN_PRIV_KEY }}
+        BIN_HOST_KEY: ${{ secrets.SFTPPLUS_BIN_HOST_KEY }}
```

Comment:
I don't think that we need this on the env... we should be able to inject them in the `run` commands without an env var.

Reply:
This is how I started, but I had problems under Windows. And then ended up using the same mechanism for all OS'es. I'll try again…